lib/extensions/active_admin/active_admin_addressable_tabs.rb

```diff
@@ -0,0 +1,8 @@
+module ActiveAdminAddressableTabs
+  # Creates tabs component which enables stimulus tabs_controller
+  def tabs(&block)
+    div 'data-controller': 'tabs' do
+      super(&block)
```

Comment:
Unfortunately, we can't add custom HTML on `tabs` level here, we can only wrap it in custom `div`.